### PR TITLE
Repair JSF itests

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

**Comment by jkopriva** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by jkopriva** [comment]
testPR

**Comment by jkopriva** [comment]
@nickboldt can you please review?

File: jsf/test-framework/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/editor/FacesConfigEditor.java [hunk 1]
```diff
@@ -56,12 +56,13 @@ private void activateEditorCTabItem(String tabItemLabel) {
...
 		activateSourceTab();
 		IEditorPart editorPart = EditorPartLookup.getInstance().getActiveEditor();
-		final org.jboss.tools.jsf.ui.editor.FacesConfigEditor fce = ((org.jboss.tools.jsf.ui.editor.FacesConfigEditor) ((EditorPartWrapper) editorPart)
+		final FacesConfigEditor fce = ((FacesConfigEditor) ((EditorPartWrapper) editorPart)
 				.getEditor());
 		ITextEditor iTextEditor = (ITextEditor) Display.syncExec(new ResultRunnable<IEditorPart>() {
 			@Override
 			public IEditorPart run() {
-				return fce.getSourceEditor();
+				// TODO get source editor
```
Comment (nickboldt) [inline jsf/test-framework/org.jboss.tools.jsf.reddeer/src/org/jboss/tools/jsf/reddeer/ui/editor/FacesConfigEditor.java]: I see a TODO here and an unreachable null on the next line.

**Comment by nickboldt** [comment]
DO NOT MERGE until code freeze is lifted next week.

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
@nickboldt  To obtain green Pull request build I updated label expression to use rhel7 only. Difference is flavor used, rhel7-devstudio-releng label has only m1.medium vs. rhel7 m1.large flavor. If you could please check and merge. Thanks

**Comment by nickboldt** [comment]
Running PRs on rhel7 instead of rhel7-devstudio-releng: +1, good idea.
CDI10 tests failing: -1, *sad trombone*.

**Comment by odockal** [comment]
Interesting that failing job is disabled, but still runs on PR... Is it even an issue when cdi10.prcheck fails on my changes to jsf part? And more further, it fails on groovy exc. while passing and empty param... Two option I see here, repair the job and make it run or remove the hook...

**Comment by odockal** [comment]
testPR

**Pull request merged**